keyboards/keebio/viterbi/rev2/config.h

```diff
@@ -19,7 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xCB10
-#define PRODUCT_ID      0x1157
+#define PRODUCT_ID      0x2157
```

Comment:
Why change this?

Reply:
It was required to get VIA to tell the Viterbi rev1 from the rev2.

This change is already present from #10167.